Doc/reference/datamodel.rst

```diff
@@ -2090,8 +2112,8 @@ method (a so-called *descriptor* class) appears in an *owner* class (the
 descriptor must be in either the owner's class dictionary or in the class
 dictionary for one of its parents).  In the examples below, "the attribute"
 refers to the attribute whose name is the key of the property in the owner
-class' :attr:`~object.__dict__`.
-
+class' :attr:`__dict__`.  The :class:`object` class itself does not
```

Comment:
Does the link to `__dict__` still work? It looks like you are reverting a change from revision bae5d81f5d1.